src/schemas/const.js

```diff
@@ -0,0 +1,3 @@
+export const REQUIRED_ERROR = 'This field is required.';
+export const MIN_VALUE_ERROR = 'This value must be greater than 0';
+export const MUST_BE_NUMBER = 'This value must a number';
```

Comment:
I think it could be a good idea to add `_ERROR` here too. Will be more consistent this way and while it's unlikely that someone might not know that this is an error message, adding the suffix will make it apparent right away.